shared/src/native-src/string.cpp

```diff
@@ -35,6 +47,21 @@ namespace shared {
 #endif
     }
 
+    std::string ToString(const GUID& uid)
```

Comment:
can you add unit tests here please ?https://github.com/DataDog/dd-trace-dotnet/blob/master/shared/test/Datadog.Trace.ClrProfiler.Native.Tests/guid_test.cpp

Reply:
Fixed in b20713cf19496c64313e59b3ad8874faad3cbb30